react/components/HomeSidebar.tsx

```diff
@@ -1,19 +1,36 @@
 import React from 'react'
 
 import { items } from '../content/SideBar'
+import SideBarItem from './SideBarItem'
+
+import VTEXBlack from './icons/VTEXBlack'
+
+interface SideBarContent {
+  text: string
+  link: string
+  articles: SideBarContent[] | []
+}
+
+let articleDepth = 0
 
 const HomeSideBar = () => (
-  <nav>
-    <ul className="list pa6 pt10">
-      {items.map(item => (
-        <li key={item.link}>
-          <a href={item.link} className="no-underline">
-            <p className="link c-muted-5">{item.text}</p>
-          </a>
-        </li>
-      ))}
-    </ul>
+  <nav className="min-h-100 br b--muted-4">
+    <VTEXBlack />
+    <ul className="list pr5 pb5 pl7 pt9">{getArticles(items, articleDepth)}</ul>
   </nav>
 )
 
+function getArticles(chapterList: SideBarContent[], depth: number): any {
+  return chapterList.map((chapter: SideBarContent) => (
```

Comment:
To fix the `li` inside `li` change getArticles to a React component and move the `<ul>` inside of it.

Reply:
That did it! Thanks! =D
The error about duplicate keys was actually because of the mocked data hahahah